data/section-templates.js

```diff
@@ -212,13 +212,25 @@ Add badges from somewhere like: [shields.io](https://shields.io/)
 
   `,
   },
-  {
+   {
     slug: 'authors',
-    name: 'Authors and Acknowledgement',
+    name: 'Authors',
+    markdown: `
+## Authors
+
+- [@katherinepeterson](https://www.github.com/katherinepeterson) .
+
+  `,
+  },
+  {
+    slug: 'acknowledgement',
+    name: 'Acknowledgements',
     markdown: `
-## Authors and Acknowledgement
+## Acknowledgements
 
-- [@katherinepeterson](https://www.github.com/katherinepeterson) for development and design.
+- [Awesome Readme Templates](https://awesomeopensource.com/project/elangosundar/awesome-README-templates)
+ - [Awesome README](https://github.com/matiassingers/awesome-readme)
+ - [How to write a Good readme](https://bulldogjob.com/news/449-how-to-write-a-good-readme-for-your-github-project)
```

Comment:
Why are these lines indented by one space, whereas the first item in the list is unindented?